nncf/quantization/quantize_model.py

```diff
@@ -394,6 +395,7 @@ def compress_weights(
     scale_estimation: Optional[bool] = None,
     gptq: Optional[bool] = None,
     lora_correction: Optional[bool] = None,
+    backup_precision: Optional[BackupPrecision] = BackupPrecision.INT8_ASYM,
```

Comment:
```suggestion
    backup_mode: Optional[CompressWeightsMode] = CompressWeightsMode.INT8_ASYM,
```
I would like to propose this change. It is more general and consistent with what compression mode will be used for backup operations.
cc' @AlexKoff88, @MaximProshin

Reply:
In this case we should add `CompressWeightsMode.FP`, which doesn't make sense

Reply:
If `CompressWeightsMode.FP` means that NNCF does not compress backup weights then None can be used.

@l-bat what do you think about renaming `backup_precision` -> `backup_mode` and introduce `BackupMode` or `BackupCompressWeightsMode`class:
```
class BackupMode(StrEnum):
  NONE = "none"
  INT8_SYM = "int8_sym"
  INT8_ASYM = "int8_asym"
```
cc' @AlexKoff88